<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adduser</class>
 <widget class="QDialog" name="adduser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>345</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ajout utilisateur</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Pseudo:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="inp_prenom">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}
</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Mot de passe:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="inp_psd">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="inp_nom">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}
</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="inp_pwd">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nom:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true">QLabel{
	font: 81 11pt &quot;Fira Sans Compressed ExtraBold&quot;;
}</string>
     </property>
     <property name="text">
      <string>&lt;center&gt;Les informations serviront a configure le profile des utilisateurs&lt;/center&gt; </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="inp_mail">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}
</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Niveau:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="Admin">
         <property name="text">
          <string>Utiliseteur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Root">
         <property name="text">
          <string>Root</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(0, 113, 0);
	height: 25px;
	width: 80px;
	font: 81 italic 12pt &quot;Fira Sans Compressed ExtraBold&quot;;
}

QPushButton:pressed{
	background-color: rgb(0, 85, 0);
}</string>
     </property>
     <property name="text">
      <string>Confirmer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Prenom(s):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Adresse mail:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
